src/xbot/common/subsystems/BasePoseSubsystem.java

```diff
@@ -47,8 +47,7 @@
     
     private BooleanProperty rioRotated;
     
-    @Inject
-    public PoseSubsystem(WPIFactory factory, XPropertyManager propManager) {
+    public BasePoseSubsystem(WPIFactory factory, XPropertyManager propManager) {
```

Comment:
Ah - I neglected to actually override the method and call updatePose(). Added that now.